### PR TITLE
Add Usability of Escritorio

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Implement usability of Escritorio.

## Steps to test these changes

1. !additem Escritorio
2. Equip, and use
3. Receive Cone Calamary

### CODE REVIEW

File: sql/item_usable.sql [hunk 1]
```diff
@@ -2332,6 +2332,7 @@ INSERT INTO `item_usable` VALUES (19204,'fiendtrapper',20,0,0,0,12,10,30,0);
 INSERT INTO `item_usable` VALUES (19246,'moggiebag',1,3,0,0,8,10,259200,0);
 -- INSERT INTO `item_usable` VALUES (20533,'worm_feelers_+1',1,1,55,0,1,30,86400,0); -- item's lua still needs made
 -- INSERT INTO `item_usable` VALUES (20568,'wind_knife_+1',4,1,0,0,1,30,600,0); -- untested
+INSERT INTO `item_usable` VALUES (20953,'escritorio',1,1,55,0,1,30,86400,0); -- Dispenses: Cone Calamary, needs animation verified.
```
Comment (TeoTwawki) [inline sql/item_usable.sql]: same animation as arthro hat and most other dispense items - I just checked. that includes moggiebag and all other mog themed items that dispense gil.

Comment (TeoTwawki) [reply sql/item_usable.sql]: if none of the other items I just referenced has a correct animation let me know

should look like:
1. player scratches butt a few seconds
2. player lifts arm up like Link obtaining the master sword
3. a little yellow circle "pops" where their hand is and disappears

Comment (CatsEyeXI) [reply sql/item_usable.sql]: I believe 117 is the one you're referring to, the arthro hat and moggie bag don't seem to work currently.

Comment (TeoTwawki) [reply sql/item_usable.sql]: Guess I need to step up the `#WarAgainstHalfMadeItems`⚔️
 
117 seems to be the "lightbulb" / "I learned something!" animation used by skill-books and Monstrosity

Comment (TeoTwawki) [reply sql/item_usable.sql]: 55 looks like the correct id:
Teo_2023 11 02_160244

**Review by Xaver-DaRed — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

**Pull request merged**